src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1133,8 +1133,8 @@ class PlacementSpec(object):
     """
     For APIs that need to specify a node subset
     """
-    def __init__(self, label=None, hosts=None, count=None):
-        # type: (Optional[str], Optional[List], Optional[int]) -> None
+    def __init__(self, label=None, hosts=None, count=None, all_hosts=False):
```

Comment:
Doesn't this belong **into** `HostPlacementSpec`?

Reply:
That was my other thought... we could force all host to have a label called ``all`` (or whatever), so that you can do ``label:all``.  I'm not sure if it'll be confusing to see all hosts with an 'all' label all the time, though.

```
HOST ADDR       LABELS 
dael 10.3.64.25 all    
gnit 10.3.64.23 all    
```

Reply:
what about simply omitting it when returning the hosts?

Reply:
So, just making ``label:all`` magic?  That would work, but it's weird because there isn't actually a label.

Right now I have it interpreting ``*`` as all_hosts=true...